tfhe/src/integer/server_key/radix_parallel/vector_find.rs

```diff
@@ -112,7 +114,7 @@ impl ServerKey {
             // Thus, in that case, the returned value is always 0 regardless of ct's value,
             // but we still have to see if the input matched something
             (
-                self.create_trivial_radix(0, ct.blocks.len()),
+                self.create_trivial_radix(0, num_blocks_to_represent_values),
                 BooleanBlock::new_unchecked(
```

Comment:
maybe a check in tests to verify this ?

Reply:
what check ?

Reply:
the number of blocks of the output is what we expect

Reply:
and not the number of blocks of the input

Reply:
Ah, I did not see the "in tests" part 🥶